app/controllers/media_controller.rb

```diff
@@ -69,4 +70,10 @@ def redirect_to_current_filename
       only_path: true,
     )
   end
+
+  def temporary_redirect?
+    false
+  end
+
+  def perform_temporary_redirect; end
```

Comment:
We can remove this method since it shouldn't get called if `temporary_redirect?` is `false`. I think it's better anyway for an error to happen if `temporary_redirect?` was changed to `true` without implementing said method.